.travis.yml

```diff
@@ -19,4 +19,4 @@ deploy:
   local_dir: build
   target-branch: master
   on:
-    branch: develop
\ No newline at end of file
+    branch: develop
```

Comment:
that is the best change ever

Reply:
yeah, I even dunno what happens here ^^,